core/controllers/profile.py

```diff
@@ -596,12 +596,24 @@ class UrlHandler(base.BaseHandler):
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'current_url': {
+                'schema': {
+                    'type': 'basestring'
+                 },
+                 'default_value': None
+             }
+        }
+    }
+
     @acl_decorators.open_access
     def get(self):
         if self.user_id:
             self.render_json({'login_url': None})
         else:
-            if self.request and self.request.get('current_url'):
+            if self.request.get('current_url'):
```

Comment:
If you remove the default `None`, you can also get rid of this `if`

Reply:
Hello @vojtechjelinek I have been working on `profile_test.py` after removing `'default_value': None` to make the tests run succesfully and after the recent commit, I have found that the coverage is not 100% and I have made the some changes to make it 100% and checked the coverage by using `python -m scripts.run_backend_tests --test_target=core.controllers.profile_test --generate_coverage_report` but I got the coverage of many other files as shown.
![Report_with_100](https://user-images.githubusercontent.com/78922463/154554647-e0a6cb7d-a921-47ab-abfd-c582479f2ccc.png)
Now, My question is since I have worked only on profile_test.py, I have to cross check the coverage of the same file and it is 100% but I am getting this error that coverage is not 100%. 
![image](https://user-images.githubusercontent.com/78922463/154554989-d27576fd-0f11-44f6-8967-e797afa707a1.png)
So, Can you please tell me what I have to do to resolve this error ?
 Thank you

Reply:
As you only run the backend tests for the _profile.py_ you don't get coverage for the other files, but that is fine. Push the changes and let the CI checks run.

Reply:
Okay @vojtechjelinek I will push the changes and yes I have the same doubt that why I got the report of other files when I strictly mentioned the file which I wanted to test and asked the same doubt in the Gitter too

Reply:
@vojtechjelinek @seanlip PTAL

Reply:
This is not done, as I said you can remove the if here. Because now this is validated using schema.

Reply:
If you remove it then we should still have the test as the error now should be raised by the schema validation.